generator/cmd/sidekick.go

```diff
@@ -28,6 +28,14 @@ func main() {
 }
 
 func root() error {
+	var projectRoot = flag.String("project-root", "", "the root of the output project")
```

Comment:
nit: eventually I would move this tree of files into a directory called sidekick. This makes it easier for people to install the program with a good name. If you run `go install github.com/googleapis/google-cloud-rust/generator/cmd` today it will install a binary on your machine named `cmd`.

Reply:
Ack will do.

Reply:
(in a future PR)